_data/redirects.yaml

```diff
@@ -45,6 +45,8 @@
 # uni
 - source: /guild
   destination: https://www.guildofstudents.com/studentgroups/societies/css/
+- source: /guild/admin
```

Comment:
my concern was that you weren't going to do this (i.e. hide it in a subdirectory) but you have. So I'm satisfied

Reply:
It’s hidden behind a layer of authentication so it doesn’t really matter anyway

Reply:
it would be clunky to not put it one layer behind